replication/README.md

```diff
@@ -366,6 +366,15 @@ message GetVolumeReplicationInfoResponse {
   // Holds the last sync time.
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
+  // Holds the last sync duration.
+  // This duration is in seconds.
+  // This field is REQUIRED.
+  int64 last_sync_duration = 2;
+  // Holds the last sync bytes.
+  // Represents number of bytes transferred
+  // with the last synchronization.
+  // This field is REQUIRED.
+  int64 last_sync_bytes = 3;
```

Comment:
Please add a detailed explanation about this field and why this is a required field, not an optional one?